json/tbd_seb_n4.json

```diff
@@ -0,0 +1,67 @@
+{
+  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "description": "testing TBD JSON for seb.osm",
+  "psis": [
+    {
+      "id": "bad",
+      "fenestration": 1.1
+    },
+    {
+      "id": "good",
+      "parapet": 0.5,
+      "rimjoist": 0.9
+    },
+    {
+      "id": "compliant",
+      "rimjoist": 0.3,
+      "parapet": 0.325,
+      "fenestration": 0.35,
+      "concave": 0.45,
+      "convex": 0.45,
+      "balcony": 0.5,
+      "party": 0.5,
+      "grade": 0.45
+    }
+  ],
+  "khis": [
+    {
+      "id": "column",
+      "point": 0.0
+    },
+    {
+      "id": "support",
+      "point": 0.0
+    }
+  ],
+  "edges": [
+    {
+      "psi": "bad",
+      "type": "fenestration",
+      "surfaces": [
+        "Entryway  Wall 5",
+        "Sub Surface 8"
+      ]
+    }
+  ],
+  "surfaces": [
+    {
+      "id": "Entryway  Wall 5",
+      "khis": [
+        {
+          "id": "column",
+          "count": 3
+        },
+        {
+          "id": "support",
+          "count": 4
+        }
+      ],
+      "psi": "(non thermal bridging)"
```

Comment:
Is `(non thermal bridging)` a special keyword?  Could you just make a new `psi` set called `perfect` with no derating and reference that instead of using a keyword?

Reply:
"(non thermal bridging)" is one of the 5 default PSI sets hardcoded in psi.rb, and so currently available in the TBD Measure PSI set pull-down options. With the input JSON file, anyone could add their own flavour, e.g. "perfect" with "0.000" W/K per m for each type.